local_docker_development/troubleshooting.md

```diff
@@ -290,6 +290,17 @@ If you are running the Windows VM in VirtualBox, you can configure it to use the
 
 Replace `"IE11 - Win10"` with the name of your VM. This will allow the VM to resolve and connect directly to your `http://*.docker.amazee.io` services running in cachalot.
 
+#### For pygmy
+
+An additional step is required if you use pygmy. Domains have to be added to Windows `hosts` file. They should point to the gateway IP address.
+
+To get the gateway IP, run `ipconfig` in Windows terminal, and search for `Default Gateway` in the output.
+
+Example `hosts` file contents:
+
+    10.0.2.2 my-local-website.com.docker.amazee.io
+    10.0.2.2 localhost
```

Comment:
@Leksat 
I think that line is too much and could cause some issues, can you remove it?

Reply:
@Schnitzel ok, removed.